### ISSUE TITLE
Documentation: Requires Internet

### ISSUE THREAD

**Issue by RubenSandwich** [ConversantLabs/SayKitSDK#3]
SayKit required a internet access, yet this is not found anywhere in the documentation.

**Comment by Quij** [comment]
Good point! Requiring an internet connection is standard for many speech tools, including SayKit, but we never mention that in the docs. We’ll add it to our next documentation update.
